docs/install.md

```diff
@@ -5,78 +5,99 @@ feature_maturity: stable
 enterprise: 'no'
 ---
 
-Spark is available in the Universe and can be installed by using either the web interface or the DC/OS CLI.
+Spark is available in the Universe and can be installed by using either the GUI or the DC/OS CLI.
 
-##  <a name="install-enterprise"></a>Prerequisites
+**Prerequisites:**
 
-- Depending on your security mode in Enterprise DC/OS, you may [need to provision a service account](https://docs.mesosphere.com/service-docs/spark/spark-auth/) before installing Spark. Only someone with `superuser` permission can create the service account.
-	- `strict` [security mode](https://docs.mesosphere.com/1.9/installing/custom/configuration-parameters/#security) requires a service account.  
-	- `permissive` security mode a service account is optional.
-	- `disabled` security mode does not require a service account.
+- [DC/OS and DC/OS CLI installed](https://docs.mesosphere.com/1.9/installing/).
```

Comment:
this is the only way to currently have the doc links work both in the repo, and after imported, on docs.mesosphere.com.